### PR TITLE
Update macvtap docs

### PR DESCRIPTION
**Pull request by maiqueb**
Reacting after user's feedback.

### CODE REVIEW

File: docs/virtual_machines/interfaces_and_networks.md [hunk 1]
```diff
@@ -922,6 +922,10 @@ instructions, in order to activate the `Macvtap` feature gate (case sensitive).
 #### Limitations
 
 - Live migration is not seamless, see [issue #5912](https://github.com/kubevirt/kubevirt/issues/5912#issuecomment-888938920)
+- On [KinD](https://github.com/kubernetes-sigs/kind) clusters, the user needs to
```
Comment (EdDev) [inline docs/virtual_machines/interfaces_and_networks.md:925]: This does not seem to be a limitation of the macvtap binding, it is a limitation of using it in a lab with kind.

Can you add it as a note warning? Although we place to remove this text in the near future, so I do not mind much on what is written here.

File: docs/virtual_machines/interfaces_and_networks.md [hunk 2]
```diff
@@ -953,11 +957,11 @@ data:
   DP_MACVTAP_CONF: |
     [
         {
-            "name"     : "dataplane",
-            "master"   : "eth0",
-            "mode"     : "bridge",
-            "capacity" : 50
-        },
+            "name"       : "dataplane",
+            "lowerDevice": "eth0",
```
Comment (EdDev) [inline docs/virtual_machines/interfaces_and_networks.md:961]: On line 950 you mention `master`.

File: docs/virtual_machines/interfaces_and_networks.md (hunk 1 above)
Comment (maiqueb) [reply docs/virtual_machines/interfaces_and_networks.md]: Done.

File: docs/virtual_machines/interfaces_and_networks.md (hunk 2 above)
Comment (maiqueb) [reply docs/virtual_machines/interfaces_and_networks.md:962]: Done.

**Review by EdDev — APPROVED** [review]

**Pull request merged**